src/webserver/WebServer.jl

```diff
@@ -219,7 +219,11 @@ function run(host, port::Integer; launchbrowser::Bool = false)
     println("Press Ctrl+C in this terminal to stop Pluto")
     println()
     
-    launchbrowser && @warn "Not implemented yet"
+    if launchbrowser
+        Sys.islinux() && Base.run(`xdg-open $address`)
```

Comment:
This does not work on WSL - is WSL2 different? Did you install `xdg-open` yourself?

Reply:
No I have not installed xdg-open I will check again on a fresh distro

Reply:
Try running `xdg-open https://github.com/` in WSL2

Reply:
> Try running `xdg-open https://github.com/` in WSL2

This works fine for me in WSL2. I you don't put http/https then it can not open.

Reply:
I found this for WSL Legacy: https://github.com/sindresorhus/open/blob/master/index.js#L71

```julia
Base.run(`cmd.exe /s /c start "" /b "https://github.com/"`)
```

It works on my machine :)

Reply:
The same command also works for Windows

Reply:
if you want to just find wls1 we can do it https://github.com/microsoft/WSL/issues/4555#issuecomment-536862561 because using calling cmd from wsl show the following log
```
'\\wsl$\Ubuntu-18.04\home\mgautam\.julia\dev\Pluto'
CMD.EXE was started with the above path as the current directory.
UNC paths are not supported.  Defaulting to Windows directory.
```